tests/integration/helpers/cluster.py

```diff
@@ -1436,6 +1444,16 @@ def setup_minio_cmd(self, instance, env_variables, docker_compose_yml_dir):
 
     def setup_azurite_cmd(self, instance, env_variables, docker_compose_yml_dir):
         self.with_azurite = True
+        env_variables["AZURITE_PORT"] = str(self.azurite_port)
+        env_variables[
+            "AZURITE_STORAGE_ACCOUNT_URL"
+        ] = f"http://azurite1:{env_variables['AZURITE_PORT']}/devstoreaccount1"
+        env_variables["AZURITE_CONNECTION_STRING"] = (
+            f"DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;"
+            f"AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;"
```

Comment:
If a string doesn't have interpolation no need for `f` before, but nothing bad will happen with it.

Reply:
indeed, for uniformity I used it here too.

Reply:
Not sure why the pylint does not complain yet. Maybe it's because of the outdated version.

In the recent versions f-strings without parameters are considered as w-level.